### ISSUE TITLE
MixSIAR error loading due to defunct packages

### ISSUE THREAD

**Issue by shwalters** [brianstock/MixSIAR#261]
Hello,

I am unable to install the MixSIARgui library.  The error states "ERROR: dependencies 'gWidgets', 'gWidgetsRGtk2' are not available for package 'MixSIARgui'".  I believe this is because 'gWidgets' has been replaced with 'gWidgets2' and 'gWidgetsRGtk2' has been similarly replaced with 'gWidgets2RGtk2'. I have installed these newer packages as well as 'RGtk2' and 'devtools'.

Can someone please advise?

Thanks!
SHW

**Comment by brianstock-NOAA** [comment]
Yes, I had to split off the GUI into a separate package for `MixSIAR` to remain on CRAN because of this issue. Please see the [`MixSIARgui`](https://github.com/brianstock/MixSIARgui) package and install instructions on the README or [here](https://github.com/brianstock/MixSIARgui/issues/2). Other experiences [here](https://github.com/brianstock/MixSIARgui/issues/1).

**Comment by shwalters** [comment]
Thanks Brian,

I finally got things to work, and it turned out to simply be an issue of using the 32-bit vs 64-bit version of R.

I had been able to install the deprecated version of 'gWidgets', but not that of 'gWidgetsRGtk2'.  Without both I obviously could not install 'MixSIARgui'.  I tried reinstalling the RGtk2 package several times, going through the GTK+ install each time as well. Regardless, I kept getting the error: "**Error: package or namespace load failed for 'RGtk2'....**".

Then I noticed in the error output kept say this as well: "**unable to load 'C:/Users/{me}/Documents/R/win-library/4.0/RGtk2/libs/i386/RGtk2.dll':**"
This seems to be the same for errors several people have posted.

Although, I had this .dll right in this location, the install process was not able to use it. I then realized that even though I was running the 64-bit version of R and had the 64-bit version of the RGtk2.dll as well, gWidgetsGRtk2 was trying to install with the i386 (32-bit) version anyway.  

So, I was able to solve the issue by first following your prescribed [steps](https://github.com/brianstock/MixSIARgui/issues/2) thru step 6. Then I simply switched the R version that Rstudio was using to be the current 32-bit version (4.0.3 in my case) and restarted the R session.  I then installed the 32-bit GTK+ by simply loading library(RGtk2) in this version of R.  From that point I had no problem finishing the rest of your steps (7 thru 11). The 'gWidgetsRGtk2' finally installed fine! The 'MixSIARgui' package then installed, and I could run the 'mixsiar_gui' function. Note: the 'mixsiar_gui' function works fine when the R version is switched back to 64-bit; it was just an install issue.

I'll also post this in the MixSIARgui #1 thread.

Cheers,
SHW
